tests/pom.xml

```diff
@@ -1172,7 +1164,7 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                     <env>
                       <LOGGING_CONFIG>file:///etc/hono/logback-spring.xml</LOGGING_CONFIG>
                       <SPRING_CONFIG_LOCATION>file:///etc/hono/</SPRING_CONFIG_LOCATION>
-                      <SPRING_PROFILES_ACTIVE>${logging.profile},embedded-cache,enable-device-connection-endpoint</SPRING_PROFILES_ACTIVE>
+                      <SPRING_PROFILES_ACTIVE>${messaging.profile}${logging.profile},embedded-cache,enable-device-connection-endpoint</SPRING_PROFILES_ACTIVE>
```

Comment:
The `messaging.profile` property doesn't get set anymore since this change from today:
https://github.com/eclipse/hono/commit/b42be3c40bd06ab74f8591453f8583e7cfd0012f#diff-4e04c04a0f54bbc17cf44b992392023397f34126f2b6f00bd7f8c3f66b87d452R343

It looks to me like the `${messaging.profile}` property should be removed altogether and the Command Router application.yml should be adapted like here:
https://github.com/eclipse/hono/commit/b42be3c40bd06ab74f8591453f8583e7cfd0012f#diff-de7239e1c07ef4513a2e3bde4da5af34ae680a9103cac7d92039a4f120da0489R79

Reply:
Thanks for pointing this out. That change was made in parallel and I was not aware of it.

Reply:
I adapted the PR to the changes on master.